using-datapower-in-kubernetes/getting-started-datapower-in-gke.md

```diff
@@ -0,0 +1,350 @@
+
+This tutorial uses the Google Container Engine to provide a hosted Kubernetes environment. Naturally,
+you can use any Kubernetes environment of your choosing. As such, we will mark the sections pertaining to
+GKE with an "(Optional)" designation.
+
+## Tools we will use
+* Docker
+    * We use Docker to package, distribute and run 
+    * https://www.docker.com
+
+* IBM DataPower Geteway for Docker
+    * Available as a Docker image in DockerHub
+    * https://hub.docker.com/r/ibmcom/datapower/
+    * https://developer.ibm.com/datapower/
+
+* Kubernetes
+    * Handle the heavy lifting of orchestrating our application
+    * http://kubernetes.io
+
+* Google Container Engine (GKE) (Optional)
+    * A hosted Kubernetes service
+    * https://cloud.google.com/container-engine
+
+* Google Cloud Shell (Optional)
+    * A shell environment for managing Google Cloud Platform resources
+    * https://cloud.google.com/shell/docs/
+    
+## Setting up our Environment
+
+### Create  Google Cloud Platform (GCP) account (Optional)
+In order to best follow along with this tutorial, you will want access to a Google Cloud Platform account.
+You can use an existing account or sign up for a free trial 
+[here](https://console.cloud.google.com/freetrial "Create a Google Cloud Platform account").
+
+### Create a GCP Project (Optional)
+You may use the default project, an existing project, or create a new one as described
+ [here](https://support.google.com/cloud/answer/6251787 "Create a GCP project").
+
+### Enable Cloud resources and Cloud Shell (Optional)
+You will need to enable the following in the [Google API Console](https://console.developers.google.com/)
+* __Compute Engine API__
+* __Container Engine API__
+
+![Google API Console](media/gcp_api_manager.png)And familiarize yourself with the Google Cloud Shell which we will use to manage our
+Google Cloud Platform resources. You can find a getting started for the Google Cloud Shell 
+[here](https://cloud.google.com/shell/docs/quickstart "Google Cloud Shell Getting Started").
+Once ready, head over to your project page in the Google Cloud Platform web console and
+start a Google Cloud Shell session.
+
+![Activate Google Cloud shell](media/gcp_activate_gcs.png)
+
+To avoid having to specify the --zone flag when required, you can set the region with:
+    
+```
+$ gcloud config set compute/zone <zone>
+```
+To see valid choices for `<zone>`, run:
+```
+$ gcloud compute zones list
+```
+
+### Setting Up a Kubernetes cluster with GKE (Optional)
+
+GKE provides a hosted Kubernetes service that can be customized. We will leverage Kubernetes to handle the complexity 
+associated with orchestrating application containers.
+
+To provision a GKE cluster, open up your project in Google Cloud Platform and start an instance of the Google Cloud Shell.
+Next, run the following command to create a GKE cluster of 3 nodes for use in this tutorial:
+
+```
+$ gcloud container clusters create k8s-cluster
```

Comment:
@jefurbee Yes, that is the default and you can naturally specify a value if the default is not satisfactory